src/sentry/apidocs/parameters.py

```diff
@@ -305,6 +305,14 @@ class ProjectParams:
         description="Activate or deactivate the client key.",
     )
 
+    IS_BOOKMARKED = OpenApiParameter(
+        name="isBookmarked",
+        location="query",
+        required=False,
+        type=bool,
+        description="Enables setting the bookmark flag when the request is invoked with a user context.",
```

Comment:
Is this user facing? If so let's make it more user friendly, like: bookmarks a project ?

Reply:
Changed to `Enables starring the project within the projects tab.`